docs/source/generated/paasta_tools.rst

```diff
@@ -88,7 +88,7 @@ Submodules
    paasta_tools.run-paasta-api-in-dev-mode
    paasta_tools.run-paasta-api-playground
    paasta_tools.secret_tools
-   paasta_tools.setup_istio_mesh
+   paasta_tools.setup_namespace_services
```

Comment:
Just to clarify, this refers to the SmartStack namespace and not a K8s namespace, correct? Since "namespace" is a bit overloaded here, would you be open to renaming this to registration to avoid ambiguity?